src/node_crypto.cc

```diff
@@ -3583,7 +3583,7 @@ SignBase::Error Sign::SignFinal(const char* key_pem,
   if (pkey == nullptr)
     goto exit;
 
-  if (EVP_SignFinal(&mdctx_, *sig, sig_len, pkey))
+  if (EVP_SignFinal(&mdctx_, *sig, sig_len, pkey) && 0 == ERR_peek_error())
```

Comment:
But in what situations does `EVP_SignFinal` return 1 and `ERR_peek_error()` is not zero?